scripts/04-find-patterns.py

```diff
@@ -1,92 +0,0 @@
-# The MIT License (MIT)
```

Comment:
@lyriccoder why is this file deleted?

Reply:
This file has is not from this merge. I've committed it accidentally, the main fix is just adding an encoding